### PR TITLE
feature/PEER-822-4 : Clickable tags for the single communities 

### PR DESCRIPTION
**Pull request by Dmitrijg24k**

### CODE REVIEW

File: app/containers/Questions/Header.jsx [hunk 1]
```diff
@@ -160,42 +274,94 @@ export const Header = ({
...
+          <TagFilterContainer>
+            <ScrollContainer>
+              <div className="df mt-md-3 mt-sm-0">
+                {tags.map((tag) => {
+                  const removeTagFilter = (removedTag) => {
+                    const searchParams = new URLSearchParams(
+                      createdHistory.location.search,
+                    );
+                    const searchParamsTags = searchParams.get('tags');
+                    const allTags = searchParamsTags?.split(':');
```
Comment (Lampardks) [inline app/containers/Questions/Header.jsx:329]:
```
const searchParams = new URLSearchParams(
                      createdHistory.location.search,
                    );
                    const searchParamsTags = searchParams.get('tags');
                    const allTags = searchParamsTags?.split(':');
```

This code duplicated the third part. Extract this code into a separate function.

File: app/containers/Questions/Header.jsx [hunk 2]
```diff
@@ -160,42 +274,94 @@ export const Header = ({
...
+                      searchParams.set('tags', result.join(':'));
+                    } else {
+                      searchParams.delete('tags');
+                    }
+                    createdHistory.push(
+                      `${createdHistory.location.pathname}?${searchParams}`,
+                    );
+                  };
+                  return (
+                    <TagFilter key={tag}>
```
Comment (Lampardks) [inline app/containers/Questions/Header.jsx:343]: Extract this item into a separate component.

File: app/containers/Questions/Header.jsx (hunk 1 above)
Comment (Dmitrijg24k) [reply app/containers/Questions/Header.jsx]: +

File: app/containers/Questions/Header.jsx (hunk 2 above)
Comment (Dmitrijg24k) [reply app/containers/Questions/Header.jsx]: +

**Review by Lampardks — APPROVED** [review]

**Pull request merged**